add CSS file with new HTML files

Rafia2019 opened the pull request.

Rafia2019 committed:
add CSS file with new HTML files

axamei reviewed (changes requested):
At first glance, you're relying heavily on the `position` property and that is going to be really messy. I recommend removing them and simplifying your code. Work on the content wrapper and hero image on the index page first and we can go back and forth about the other sections.

Rafia2019 committed:
check new changes  in the CSS file and add them

First, I have added dropdown menu under "Artwork".  I can see the navigation bar and it is working in my screen. Second, Could not allign the hero image with the navigation bar( ). Third, the text box does not stay center to it's position when I resize the browser screen.
